src/__spec_helper__/mock-dom-rect.ts

```diff
@@ -0,0 +1,28 @@
+const getDOMRect = (width: number, height: number) => ({
```

Comment:
**suggestion(non-blocking)**: what do you think about adding a return type here?

```suggestion
const getDOMRect = (width: number, height: number): DOMRect => ({
```